components/README.md

```diff
@@ -0,0 +1,127 @@
+[TOC]
+
+
+
+# Dialogs
+
+Dialogs are a standard way to inform the user about some result, giving the user option to react to it.
+
+<details>
+    <summary>Code snippet</summary>
+    <br>
+https://github.com/appunite/Loudius/blob/8e616a9a05e668d11425a40c9f452eb3091dcc2a/components/src/main/java/com/appunite/loudius/components/components/LoudiusDialog.kt#L25-L48
+</details>
+| Variant                                                      | Snapshot                                                     |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| **Standard** <br />This component gives the user option to confirm the information. | <img src="src/test/snapshots/images/com.appunite.loudius_PaparazziShowkaseTests_preview_tests[Default Group-LoudiusDialogSimplePreview,1,light].png" alt="alt text" style="zoom: 80%;" /> |
+| **Advanced**<br />This component gives  the user the choice out of two options, usually dismiss or confirmation. | <img src="src/test/snapshots/images/com.appunite.loudius_PaparazziShowkaseTests_preview_tests[Default Group-LoudiusDialogAdvancedPreview,1,light].png" alt="alt text" style="zoom: 80%;" /> |
```

Comment:
chore: the current solution doesn't work: https://github.com/appunite/Loudius/blob/chore/design_system_docs/components/README.md